sky/cli.py

```diff
@@ -208,15 +208,53 @@ def _interactive_node_cli_command(cli_func):
                               default=False,
                               required=False,
                               help='Skip confirmation prompt.')
+    idle_autostop = click.option('--idle-minutes-to-autostop',
+                                 '-i',
+                                 default=None,
+                                 type=int,
+                                 required=False,
+                                 help=('Automatically stop the cluster after '
+                                       'this many minutes of idleness, i.e. '
+                                       'no running or pending jobs in the '
+                                       'cluster\'s job queue. Idleness starts '
+                                       'counting after setup/file_mounts are '
+                                       'done; the clock gets reset whenever '
+                                       'there are running/pending jobs in the '
+                                       'job queue. If not set, the cluster '
+                                       'will not be auto-stopped.'))
+    retry_until_up = click.option('--retry-until-up',
+                                  '-r',
+                                  is_flag=True,
+                                  default=False,
+                                  required=False,
+                                  help=('Whether to retry provisioning '
+                                        'infinitely until the cluster is up '
+                                        'if we fail to launch the cluster on '
+                                        'any possible region/cloud due to '
+                                        'unavailability errors.'))
+    region_option = click.option('--region',
+                                 default=None,
+                                 type=str,
+                                 required=False,
+                                 help='The region to use.')
+    zone_option = click.option('--zone',
+                               default=None,
+                               type=str,
+                               required=False,
+                               help='The zone to use.')
```

Comment:
Instead of duplicating the options here, can we directly use the `_TASK_OPTIONS` or split the `_TASK_OPTIONS` to `_TASK_OPTIONS` and `_CLUSTER_OPTIONS`, so that both `launch` and `{cpu,gpu,tpu}node` can share the similar options (including the cloud, instance_type, ... options above)?

Reply:
One drawback is that the descriptions in `_TASK_OPTIONS` refer to config in the YAML, but `sky {cpu,gpu,tpu} node` don't use the YAML.

Reply:
Good point! Do you think it is possible to move the construction of the `_TASK_OPTIONS` to a function?
```
def _get_task_options(has_yaml: bool) -> List[click.Option]
```

Reply:
Sure, let me try this.

Reply:
@Michaelvll I have a question. Is `If specified, overrides the "resources.cloud" config.` referring to the yaml config? I thought it was, but there are other help messages explicitly refer to the yaml config (like `Overrides the "num_nodes" config in the YAML if both are supplied.`)

Reply:
Yes, they should all refer to the YAML config. Great catch! We should make them consistent, e.g. `If specified, overrides the "resources.cloud" config in the YAML.` and ``If specified, overrides the "num_nodes" config in the YAML.`